src/pudl/constants.py

```diff
@@ -1633,6 +1633,7 @@
 # The full set of years we currently expect to be able to ingest, per source:
 working_years = {
     'eia860': tuple(range(2009, 2020)),
+    'eia860m': ["2020-08"],
```

Comment:
yes, I was wondering about this. I suppose it would be better to have `'eia860m': tuple(['2018-10'])` here, but maybe it doesn't make sense at all to have this year-month inside of `working_years` because it is not really a year at all.

Reply:
I don't think there's any reason it needs to be a collection at all, is there? If it's only ever allowed to be a single value, we should just store the single value. But anything that's being stored and used as an argument for other functions, like in the other datasets here, needs to be non-mutable (i.e. tuple instead of list) because otherwise the arguments can get accidentally changed in one place, and that change will appear everywhere the value is referenced since it has a scope outside of any of the functions.

Reply:
I went ahead and changed `working_years` to `working_partitions` and within each data set is another dictionary with the name of the partition as the keys and the disparate partition types as the values.